plugins/spammer/spammer.go

```diff
@@ -67,13 +105,15 @@ func doSpam(shutdownSignal <-chan struct{}) {
 	for _, tx := range b {
 		txTrits, _ := transaction.TransactionToTrits(&tx)
 		if err := gossip.Processor().CompressAndEmit(&tx, txTrits); err != nil {
+			log.Infof("worker idle because CompressAndEmit err: %s", err)
+			randomSleep()
 			return
 		}
 		metrics.SharedServerMetrics.SentSpamTransactions.Inc()
 	}
 
 	durTotal := time.Since(timeStart).Truncate(time.Millisecond)
-	log.Infof("Sent Spam Transaction: #%d, TxHash: %v, GTTA: %v, PoW: %v, Total: %v", txCountValue, b[0].Hash, durGTTA.Truncate(time.Millisecond), durPOW.Truncate(time.Millisecond), durTotal.Truncate(time.Millisecond))
+	log.Infof("sent transaction: #%d, TxHash: %v, GTTA: %v, PoW: %v, Total: %v", txCountValue, b[0].Hash, durGTTA.Truncate(time.Millisecond), durPOW.Truncate(time.Millisecond), durTotal.Truncate(time.Millisecond))
```

Comment:
`sent spam transaction` is the correct comment.
Only `sent transaction` could be confusing IMHO.

Reply:
Now the log output looks like this
`0|hornet  | 2020-05-13T12:35:14+02:00   INFO    Spammer sent transaction: #958153, TxHash: KAZESYBZCY9D9CNBQJKOQDZRRKUUJM9VPGDLFNWMFZSLOKIOGSUWSYJOGUTWZWHJTTL9XFCCMYZWV9999, GTTA: 111ms, PoW: 186ms, Total: 298ms`

In the "Sent Spam Transaction" I had the feeling the Spam word was double in the final output. Plus the line was longer and wrapped more easy

Reply:
fair enough! :+1: